lib/model/frames/entity.js

```diff
@@ -27,7 +27,7 @@ class Entity extends Frame.define(
     'int4', 'int4',
     'conflictType',
     'timestamptz',
-    'timestamptz', 'timestampz',
+    'timestamptz', 'timestamptz',
```

Comment:
the comment before the class definition has became obsolete - not by this change though